_episodes/08-func.md

```diff
@@ -18,8 +18,8 @@ keypoints:
 - "Call a function using `function_name(value)`."
 - "Numbers are stored as integers or floating-point numbers."
 - "Variables defined within a function can only be seen and used within the body of the function."
-- "If a variable is not defined within the function it is used,
-   Python looks for a definition before the function call"
+- "If a variable is not locally defined within the function it is used,
+   Python looks for a global definition before the function call"
```

Comment:
I'm not positive adding "locally" and "global" improves this already rather long keypoint.
Shall we, perhaps, split it into several smaller ones?
1. Variables created outside of any function are called "global" variables.
1. Within a function, we can access global variables.
1. Variables created within a function override global variables if their names match.

Reply:
Agreed.